### PR TITLE
Implement get view from reactTag on Android and iOS.

### PR DESCRIPTION
**Pull request by jgfidelis**
Do not save app context on CameraModule as well

### CODE REVIEW

File: src/RNCamera.js [hunk 1]
```diff
@@ -195,14 +197,25 @@ export default class Camera extends React.Component<PropsType> {
...
   render() {
     const nativeProps = this._convertNativeProps(this.props);
 
     return (
       <RNCamera
         {...nativeProps}
+        ref={this._setReference}
         onMountError={this._onMountError}
-        onCameraRead={this._onCameraReady}
+        onCameraReady={this._onCameraReady}
```
Comment (sibelius) [inline src/RNCamera.js]: nice

File: ios/RN/RNCameraManager.m [hunk 2]
```diff
@@ -198,43 +192,30 @@ + (NSDictionary *)faceDetectorConstants
...
+        if (![view isKindOfClass:[RNCamera class]]) {
+            RCTLogError(@"Invalid view returned from registry, expecting RNCamera, got: %@", view);
         } else {
             [view record:options resolve:resolve reject:reject];
         }
     }];
 }
 
-RCT_EXPORT_METHOD(stopRecording)
+RCT_REMAP_METHOD(stopRecording, reactTag:(nonnull NSNumber *)reactTag)
```
Comment (sibelius) [inline ios/RN/RNCameraManager.m]: remap?

Comment (jgfidelis) [reply ios/RN/RNCameraManager.m]: Yes, when you receive a parameter you should use RCT_REMAP_METHOD.

Comment (sibelius) [reply ios/RN/RNCameraManager.m]: Great

**Pull request merged**